.gitignore

```diff
@@ -0,0 +1,2 @@
+*.elc
+
```

Comment:
Please avoid adding a trailing empty line.

Reply:
Thanks, this change is unneeded for this PR, I just remove this commit.